src/dh/clu_dh.c

```diff
@@ -212,6 +212,15 @@ int wolfCLU_DhParamSetup(int argc, char** argv)
 
     /* generate the dh parameters */
     if (ret == WOLFCLU_SUCCESS && bioIn == NULL) {
+    #ifdef HAVE_FFDHE_4096
```

Comment:
Should we also support the older method for DH pre v5 wolfSSL?

```
#ifdef HAVE_PUBLIC_FFDHE
    const DhParams* params = wc_Dh_ffdhe4096_Get();
    ret = wc_DhSetKey(&args->key->priv.dh.key,
        (byte*)params->p, params->p_len,
        (byte*)params->g, params->g_len);
#else
    ret = wc_DhSetNamedKey(&dh, WC_FFDHE_4096);
#endif
```

Reply:
Is a little spotty for support for older wolfSSL versions. To test this with v4.8.1 I used the configure `./configure --enable-crl --enable-keygen --enable-dh --enable-certreq --enable-opensslextra --enable-opensslall --enable-certgen --enable-des3 --enable-sha512 --enable-sha3 --enable-ed25519 --enable-aesctr CPPFLAGS="-DHAVE_FFDHE_4096 -DHAVE_PUBLIC_FFDHE"` and removed -Werror from the makefile since there is still a couple of warnings with old versions of wolfSSL. Then ran `./wolfssl dhparam 4096`.